pkg/pcaps/pcap.go

```diff
@@ -67,6 +73,8 @@ func NewPcap(e *trace.Event, t PcapType) (*Pcap, error) {
 }
 
 func (p *Pcap) write(event *trace.Event, payload []byte) error {
+	p.fileLock.Lock()
```

Comment:
Is there more than one writer?

Reply:
I need more tests. I was able to corrupt the pcap file without it. With the locking, I wasn't (ever after transferring 3GB of data and capturing a huge pcap). I wonder if the primitives used for the mutex (barriers) are avoiding a race of some sort (but this code isn't supposed to be parallel).

Checking...

Reply:
![image](https://user-images.githubusercontent.com/7395852/213424108-d4491f50-ed36-4c52-87be-17aa9adc93a9.png)

Looking good now. I think I got some situation with a partial flush to the file that might have been done in a later moment due to workload. Will keep an eye on pcap files and check if they're always consistent. Removing the lock for now.